apps/smart-contracts/core/helpers/index.ts

```diff
@@ -26,3 +34,91 @@ export async function createMarket(marketParams: CreateMarketParams): Promise<Cr
     hash: events[0].args.longShortHash,
   }
 }
+
+export async function baseTokenNeededForMinting(
+  collateral: Collateral | MockContract<Collateral>,
+  amount: BigNumber
+): Promise<BigNumber> {
+  const fee = amount.mul(await collateral.getDepositFee()).div(FEE_DENOMINATOR)
+  return amount.sub(fee)
+}
+
+export async function depositCapCheck(
```

Comment:
nit: name functions as actions rather than statements
```suggestion
export async function checkDepositCap(
```